src/uwtools/drivers/ioda.py

```diff
@@ -0,0 +1,77 @@
+"""
+A driver for the ioda component.
+"""
+
+from datetime import datetime
+from pathlib import Path
+from typing import List, Optional
+
+from iotaa import tasks
+
+from uwtools.drivers.jedi import JEDI
+from uwtools.strings import STR
+
+
+class IODA(JEDI):
```

Comment:
While I was testing IODA, the only thing that was different from JEDI was that it shouldn't be running the `validate_only` task. Also the `jedi.yaml` that shows up in the run directory is misleading. I subclassed JEDI here to achieve that behavior, but I didn't remove the `validate_only` task from the subclass.

Should I remove it? The internet tells me it violates the Liskov Substitution Principle, and I shouldn't do it.

Reply:
Are the DRY savings enough to justify inheritance vs duplication? If not, some duplication would be easier. If not, the "right" way to do it is to create a common ancestor class and have both `JEDI` and `IODA` inherit from it, as we did with `MPASBase`. Then only `JEDI` would implement `validate_only()`, and `IODA` could inherit all the truly common attributes.